modular_coyote/code/mob.dm

```diff
@@ -16,5 +16,8 @@
 	emote_taunt_sound = list('sound/f13npc/deathclaw/taunt.ogg')
 	aggrosound = list('sound/f13npc/deathclaw/aggro1.ogg', 'sound/f13npc/deathclaw/aggro2.ogg', )
 	idlesound = list('sound/f13npc/deathclaw/idle.ogg',)
-	death_sound = list('sound/f13npc/hellpig/hellpigdead.ogg')
+	death_sound = list('sound/f13npc/hellpig/hellpigdead.ogg',)
+ guaranteed_butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 10, /obj/item/stack/sheet/bone = 6,)
```

Comment:
Not sure if this'll cause a problem, but its on a different indentation from the rest. Just to be safe, please line it all up.